### PR TITLE
Extend routed-based navigation

### PR DESCRIPTION
**Pull request by LostCrew**
Add subviews of Estimate page:
- `/estimate/add-flight` Add flight
- `/estimate/edit-flight/:id` Edit flight
- `/estimate/checkout` Checkout
- `/estimate/success` Success

### CODE REVIEW

File: src/views/EstimateCheckout.vue [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+          paymentIntentClientSecret
+        )
+        const confirm = await this.fetchConfirm(paymentIntent)
+        this.onCheckoutResponse(confirm)
+      } else if (success) {
+        trackEvent('paymentSuccessful', { 'app.estimateID': this.estimateID })
+        if (customerID) {
+          this.$cookies.set('custID', customerID)
+        }
+        this.$router.push('/success')
```
Comment (jasongwartz) [inline src/views/EstimateCheckout.vue]: I think this needs to be changed to point to the new success URL? It doesn’t work when I test it in the preview deploy (after successful payment, I’m redirected to the 404 page).

Comment (LostCrew) [reply src/views/EstimateCheckout.vue]: Absolutely right

**Comment by LostCrew** [comment]
Currency switcher restored:

image

**Comment by jasongwartz** [comment]
I just tested this on the branch deploy - can't go through the checkout procedure (the route is updated and `flights=` is removed).

**Comment by LostCrew** [comment]
@jasongwartz doesn't it need https://github.com/neutral-af/frontend/pull/30?

**Comment by jasongwartz** [comment]
dunno - but I wasn't doing a hard page refresh, was just trying to navigate the basic flow.

but feel free to fix the merge conflicts and try

**Comment by LostCrew** [comment]
@jasongwartz Can reproduce. The router push don't preserve the current querystring but it should. Fix incoming

**Pull request merged**